Sources/SwiftFusion/Geometry/Pose2.swift

```diff
@@ -114,106 +192,78 @@ public struct Pose2: Differentiable, TangentStandardBasis, KeyPathIterable {
   }
 
   /// The derivative of `rot` satisfiying convention (2) described above.
-  @derivative(of: rot)
+  @derivative(of: Pose2Coordinate.rot)
   @usableFromInline
   func vjpRot() -> (value: Rot2, pullback: (Vector1) -> Vector3) {
     // This pullback is similar to the pullback in `vjpT`, but for the second column of the
     // matrix. See the comment in `vjpT` for more information.
     return (rot, { Self.tangentVector(w: $0, v: Vector2.zero) })
   }
+}
 
-  /// Twist coordinates, with the first component for rotation and the remaining components for
-  /// translation.
-  ///
-  /// See the documentation comment on the declaration of `Pose2` for more information.
-  public typealias TangentVector = Vector3
-
-  /// Creates a tangent vector given rotation (`w`) and translation (`v`) components.
-  public static func tangentVector(w: Vector1, v: Vector2) -> Vector3 {
-    Vector3(w.x, v.x, v.y)
+// MARK: Coordinate Operators
+public extension Pose2Coordinate {
+  /// Product of two rotations.
+  @differentiable
+  static func * (lhs: Pose2Coordinate, rhs: Pose2Coordinate) -> Pose2Coordinate {
+    Pose2Coordinate(lhs.rot * rhs.rot, lhs.t + lhs.rot * rhs.t)
   }
 
-  /// Returns the rotation (`w`) and translation (`v`) components of `tangentVector`.
-  public static func decomposed(tangentVector: Vector3) -> (w: Vector1, v: Vector2) {
-    (Vector1(tangentVector.x), Vector2(tangentVector.y, tangentVector.z))
+  /// Inverse of the rotation.
+  @differentiable
+  func inverse() -> Pose2Coordinate {
+    Pose2Coordinate(self.rot.inverse(), self.rot.unrotate(-self.t))
   }
+}
 
-  /// Moves `self` by `exp(hat(direction))`.
+extension Pose2Coordinate: ManifoldCoordinate {
   public mutating func move(along direction: Vector3) {
```

Comment:
Delete this `move`. The automatically-derived `move` function for `ManifoldCoordinate` is the correct one for it.